birdhouse/components/weaver/config/proxy/docker-compose-extra.yml

```diff
@@ -7,6 +7,6 @@ services:
       - ./components/weaver/config/proxy/conf.extra-service.d:/etc/nginx/conf.extra-service.d/weaver:ro
       # because of mounting path naming restrictions (see note in 'worker' definition),
       # we must add the custom path on top of named 'wps_outputs' volume of other birds for the proxy to expose results
-      - ${WEAVER_WPS_OUTPUTS_DIR}:/pavics-data/wps_outputs/weaver:ro
```

Comment:
Not sure what `THREDDS` has to do here for `weaver`/`proxy` service combination regarding WPS outputs.

Same comment for all other locations edited in https://github.com/bird-house/birdhouse-deploy/pull/359/commits/91af29eff88393f5a7d4c1778c8e5d90a8e8615d that are not directly related to `thredds` service.